src/OrdersList/OrdersListFilters.js

```diff
@@ -30,7 +31,7 @@ import {
 
 const applyFiltersAdapter = (applyFilters) => ({ name, values }) => applyFilters(name, values);
 
-function OrdersListFilters({ activeFilters, closingReasons, applyFilters, disabled, addresses }) {
+function OrdersListFilters({ activeFilters, closingReasons, customFields, applyFilters, disabled, addresses }) {
```

Comment:
break down into new lines if more than 3 props.
```suggestion
function OrdersListFilters({ 
   activeFilters, 
   closingReasons, 
   customFields, 
   applyFilters, 
   disabled, 
   addresses,
}) {
```